src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -123,7 +123,21 @@ void rtc6705_soft_spi_set_channel(uint16_t channel_freq)
 
 void rtc6705_soft_spi_set_rf_power(uint8_t reduce_power)
 {
-    rtc6705_write_register(7, (reduce_power ? (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK)) : PA_CONTROL_DEFAULT));
+    switch(reduce_power)
+    {
+        case 0:// 1dBm
+            rtc6705_write_register(7, 0x006A);
```

Comment:
Upon further review tx5823 and fx758 modules setting register 7 perform as expected from what is coded here. They also share the same PA chip. The La Forge VTX uses a different PA and had different results when changing anything other then MAI_5G.

So I think what I was seeing comes down to which PA is used.
Just goes to show the diversity in these VTX modules.

I did notice I was late to the conversation and apologize for any confusion I may have added.
https://github.com/betaflight/betaflight/pull/1839#issuecomment-277503675

The only thing I would suggest investigating is changing MAI_5G (5G Pre-driver output control) since it performed the same on all VTX modules I tested and might be a more reliable way to get LOW, MEDIUM, HIGH.